dev-docs/RFCs/v5.3/generic-accessor-props-rfc.md

```diff
@@ -0,0 +1,89 @@
+# Generic Accessors Micro-RFC
+
+## Background
+
+Many layers use accessors that return constants, for example:
+
+```js
+new ScatterplotLayer({
+  radiusScale: 100,
+  getPosition: d => d.position,
+  getRadius: d => 1,
+  getColor: d => [255, 200, 0]
+});
+```
+
+The curent implementation to handle this use case has several problems:
+
+* A large buffer is created and filled with the same value. This can be much more efficient if we switch to using generic attribute.
+* Update triggers are required to change these constants, which has been a pain point for beginners.
+
+
+## Proposal
+
+### Option 1: New Uniform Props
```

Comment:
Nit: Unless we are still uncertain, I prefer a single "Proposal" saying what we want to do, followed by an "Alternatives" section to show we have considered other options.